CHANGES.md

```diff
@@ -21,10 +21,22 @@ To be released.
 
 ### Behavioral changes
 
+ -  `Swarm<T>.PreloadAsync()` and `Swarm<T>.StartAsync()` became to download
```

Comment:
Is this related to this PR?

Reply:
It was reverted in #792. I will remove lines related to it.